vscode_extension/src/AacLanguageServer.ts

```diff
@@ -30,8 +30,8 @@ export class AacLanguageServerClient {
 
     private assertAacToolIsAvailable(): void {
         const pythonPath = this.getConfigurationItemFile("pythonPath");
-        this.assertCorrectPythonVersionIsInstalled(pythonPath);
         this.assertCorrectAacVersionIsInstalled(pythonPath);
+        this.assertCorrectPythonVersionIsInstalled(pythonPath);
```

Comment:
Is there a reason to change the order here?

```suggestion
        this.assertCorrectPythonVersionIsInstalled(pythonPath);
        this.assertCorrectAacVersionIsInstalled(pythonPath);
```

Reply:
No, it's an artifact of my removing then putting it back.